cluster-autoscaler/clusterstate/clusterstate.go

```diff
@@ -501,6 +501,14 @@ func (csr *ClusterStateRegistry) IsNodeGroupScalingUp(nodeGroupName string) bool
 	return found
 }
 
+// HasNodeGroupStartedScaleUp returns true if the node group has started scale up.
+// Difference between IsNodeGroupScalingUp() is, this function doesn't check whether
+// there are any upcoming nodes for the node group.
+func (csr *ClusterStateRegistry) HasNodeGroupStartedScaleUp(nodeGroupName string) bool {
+	_, found := csr.scaleUpRequests[nodeGroupName]
```

Comment:
Node group requests are cleared in https://github.com/kubernetes/autoscaler/blob/763c0e85492f57a5b51936767a95557c36fcc40e/cluster-autoscaler/clusterstate/clusterstate.go#L258, what semantics exactly are you looking for here?